dataflow/build.gradle

```diff
@@ -55,7 +55,12 @@ createDataflowShaded('nullaway')
 createDataflowShaded('errorprone')
 
 /**
- * Create a task to run a dataflow analysis demo
+ * Create a task to run a dataflow analysis testcase
+ * To add a dataflow analysis case, in normal cases you only
+ * need to add your test code directory and the classname and
+ * method {org.checkerframework.dataflow.util.DataflowTester.performTest}
+ * will receive your analysis instance. Go org.checkerframework.dataflow.util.DataflowTester
+ * to see more details if you want to make any change on it.
```

Comment:
This is my personal suggestion:
"""
Usually, to add a new dataflow analysis test, you only need to provide the directory which contains some test files and the classname of the test entrance. Please check org.checkerframework.dataflow.util.DataflowTester for more details about ...
"""